frontend/src/global.css

```diff
@@ -0,0 +1,11 @@
+html {
+  scroll-behavior: smooth;
+  scroll-padding-top: 5.5rem;
+}
+
+@media screen and (max-width: 991px) {
+  html {
+    scroll-behavior: smooth;
```

Comment:
Is this not going to still be covered by the above?

Reply:
No sir as in the mobile view the padding needed is different 

Desktop-view:- 
![Screenshot from 2024-03-01 00-48-56](https://github.com/bowtie-json-schema/bowtie/assets/69006513/eade70ff-0b57-4ce2-8b4e-55b2b82f2ac8)

Tab and Mobile view

![Screenshot from 2024-03-01 00-48-45](https://github.com/bowtie-json-schema/bowtie/assets/69006513/17cec0ed-850e-43a7-a92c-2e2f0643dada)

Reply:
The padding is different but I was asking about the `scroll-behavior` which you have in both sections, and which presumably should get inherited from the top one.

But considering that even more carefully, let's just remove that entirely, I'm not a fan of silly animations if they make things slower, let's just leave it at the default behavior and just add the padding.

Also please add a little extra padding so that borders don't end up at the top -- say, `6rem` and `19rem` look like they leave the little extra space at the top which looks a bit better I think.

Reply:
@Julian Gotcha sir.